espnet2/speechlm/module/abs_transformer.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Jinchuan Tian
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+from abc import ABC, abstractmethod
```

Comment:
what is `ABC`?
(sorry. simply I don't know this)

Reply:
abstract base class